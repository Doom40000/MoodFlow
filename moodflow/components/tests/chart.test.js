
import renderer from 'react-test-renderer';
import React from "react"
import { render, fireEvent, waitFor  } from "@testing-library/react-native"
import Chart from '../chart/chart';

describe('Chart', () => {
  it('should render the page', () => {
    const tree = renderer.create(<Chart/>).toJSON;
    expect(tree).toMatchSnapshot();
  })
  it('renders correctly', () => {
    const { getByTestId } = render(<Chart />);
    expect(getByTestId('list')).toBeTruthy();
    expect(getByTestId('listItem-Sleep quality')).toBeTruthy();
  });

})
