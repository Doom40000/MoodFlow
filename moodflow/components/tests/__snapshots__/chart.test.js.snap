// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chart should render the page 1`] = `[Function]`;
